src/BlazorWebView/src/Maui/Android/WebKitWebViewClient.cs

```diff
@@ -35,13 +37,33 @@ public override bool ShouldOverrideUrlLoading(AWebView? view, IWebResourceReques
 		{
 			// Handle redirects to the app custom scheme by reloading the URL in the view.
 			// Handle navigation to external URLs using the system browser, unless overriden.
-			var requestUri = request?.Url?.ToString();
+			if (request?.Url is { } url)
+			{
+				if (url.Scheme is "http" or "https")
+				{
+					if (TryOverrideHttpUrlLoading(view, request, url.ToString()))
```

Comment:
It's a bit odd that we stringify the URL here and then re-parse it in `TryOverrideHttpUrlLoading`. Would it work to keep it in the form of an `Android.Net.Uri` object?

I see it has to be made into a `System.Uri` if we're going to call `ExternalNavigationStarting` but that doesn't happen in most navigations (to internal URLs).